scripts/jest/ts-preprocessor.js

```diff
@@ -19,7 +19,16 @@ function compile(content, contentFilename) {
   var output = null;
   var compilerHost = {
     getSourceFile: function(filename, languageVersion) {
-      var source;
+      var source, reactRegex;
+
+      // Accomodations for backslashes in Windows file paths.
+      if (process.platform === 'win32') {
+        filename = filename.replace(/\//g, '\\');
```

Comment:
Can we use `path` APIs to do this without having to do it manually? Eg `path.normalize` (no idea if it works, just that it exists). Might be able to swing that for the regexp too (eg `new RegExp(path.join('/', '(?:React|ReactDOM)(?:\.d)?\.ts$'))`